v4/source/definitions.yaml

```diff
@@ -132,6 +132,10 @@ definitions:
         type: integer
       total_msg_count:
         type: integer
+      extra_update_at:
+        description: Removed in Mattermost 5.0 release
```

Comment:
To be pedantic, since we're keeping both the column and the mention, `Deprecated in Mattermost 5.0 release` seems a bit clearer.